src/CommonLib/aspnetcoreconfig.h

```diff
@@ -283,4 +291,7 @@ class ASPNETCORE_CONFIG : IHttpStoredContext
     BOOL                   m_fIsStandAloneApplication;
     APP_HOSTING_MODEL      m_hostingModel;
     ENVIRONMENT_VAR_HASH*  m_pEnvironmentVariables;
+    STRU                   m_struHostFxrLocation;
+    PCWSTR*                m_pcwArguments;
```

Comment:
it is better to rename it as m_ppStrArguments as it points to a array of string pointers